src/shop-client.js

```diff
@@ -271,15 +270,19 @@ const ShopClient = CoreObject.extend({
   },
 
   /**
-    * Convenience wrapper for {{#crossLink "ShopClient/update:method"}}
-    * {{/crossLink}}. Equivalent to:
+    * Updates an existing {{#crossLink "CartModel"}}CartModel{{/crossLink}} instance and saves it.
```

Comment:
so this method no longer makes an API call, correct? We should be consistent about indicating what 'saves' means.